tools/akita-cli/templates/angular/index.tpl

```diff
@@ -1,8 +1,8 @@
 export * from './{{dashCase name}}.query';
-export * from './{{dashCase name}}.store';
+export { {{ pascalCase name }}State, {{pascalCase name}}Store } from './{{dashCase name}}.store';
```

Comment:
Why not omit the export from `createInitialState`?

Reply:
You might want to use this function in a service. So, it should be exported, but only inside a store. But if you find it a rare use case, I can change it.

Reply:
Sounds ok.